corehq/messaging/scheduling/templates/scheduling/dashboard.html

```diff
@@ -0,0 +1,112 @@
+{% extends "hqwebapp/base_section.html" %}
+{% load hq_shared_tags %}
+{% load i18n %}
+
+
+{% block stylesheets %}{{ block.super }}
+<style>
+    .messaging-dashboard-indicator{
+        font-size: 20px;
+    };
+</style>
+{% endblock %}
+
+
+{% block js %}{{ block.super }}
+    <script src="{% static 'scheduling/js/dashboard.js' %}"></script>
+{% endblock %}
+
+
+{% block page_content %}
+    {% registerurl 'messaging_dashboard' domain %}
+    {# Prevent flicker by hiding and using bindingApplied to make visible #}
+    <div id="messaging_dashboard" class="container-fluid" style="display: none;" data-bind="visible: bindingApplied">
+        <div>
+            <p>
+                <span>{% trans "Last updated:" %}</span>
+                <span data-bind="text: last_refresh_time"></span>
+                <span>
+                    (<span data-bind="text: project_timezone"></span>)
+                </span>
+            </p>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">
+                    <strong>{% trans "SMS Status:" %}</strong>
+                    <strong class="text-success" data-bind="visible: is_sms_currently_allowed() && queued_sms_count() === 0">{% trans "All Caught Up" %}</strong>
+                    <strong class="text-warning" data-bind="visible: is_sms_currently_allowed() && queued_sms_count() > 0">{% trans "Working" %}</strong>
+                    <strong class="text-danger" data-bind="visible: !is_sms_currently_allowed()">{% trans "Paused" %}</strong>
+                    <span data-bind="visible: queued_sms_count() > 0 || !is_sms_currently_allowed()">
+                        (<span data-bind="text: queued_sms_count"></span>
+                        {% trans "SMS in the queue" %})
+                    </span>
+                </h3>
+            </div>
+            <div class="panel-body">
+                <div class="row" data-bind="visible: !is_sms_currently_allowed()">
+                    <p class="help-block">
+                        <span><i class="fa fa-info-circle"></i> {% trans "Messages will remain queued until the below conditions are cleared." %}</span>
```

Comment:
Grammar nazi super nit: Should be "the following conditions" or "the conditions below" (because "below" is a preposition not an adjective :nerd_face: ). Probably OK to leave as is, because it's obvious what it means.

Reply:
(nvm. I didn't notice this was outdated. I should have :office: and not :blowfish:)

Reply:
good point, now that you mention it "the following conditions" just sounds better: https://github.com/dimagi/commcare-hq/pull/20178/commits/d995978518a50d0d1ee0f92312908c847d2838a3